curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md

```diff
@@ -7,11 +7,13 @@ dashedName: step-8
 
 # --description--
 
-HTML <dfn>attributes</dfn> are special words used inside the opening tag of an element to control the element's behavior. The `src` attribute in an `img` element specifies the image's URL (where the image is located). An example of an `img` element using an `src` attribute: `<img src="https://www.example.com/the-image.jpg">`.
+HTML <dfn>attributes</dfn> are special words used inside the opening tag of an element to control the element's behavior. The `src` attribute in an `img` element specifies the image's URL (where the image is located).
 
-Inside the existing `img` element, add an `src` attribute with this URL:
+Here is an example of an `img` element with a `src` attribute pointing to the freeCodeCamp logo:
 
-`https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg`
+```html
+<img src="https://cdn.freecodecamp.org/platform/universal/fcc_secondary.svg">
+```
```

Comment:
The instructions for the step should not be removed.

The example should instead come before the `Inside the existing...` instructions.

Reply:
Whoops! Sorry about that, added those back.